### PR TITLE
refactoring done, test passed

### PR DESCRIPTION
**Pull request by SlavaKlepikov**

### CODE REVIEW

File: src/test/java/pages/SearchPage.java [hunk 1]
```diff
@@ -35,6 +35,10 @@ public void selectProjectQAAUTO6(String request){
         $(By.cssSelector(".criteria-selector.aui-button.aui-button-subtle.drop-arrow")).click();
         $(By.id("searcher-pid-input")).setValue(request);
         $(By.cssSelector("label[title='QAAUTO-6']")).click(); }
+    public void selectProject(String request){
```
Comment (Artem100) [inline src/test/java/pages/SearchPage.java]: Put the method on the next line (press 'Enter')

File: src/test/java/pages/SearchPage.java [hunk 2]
```diff
@@ -69,4 +73,11 @@ public boolean atRequiredPage()
     { Assert.assertEquals(title(), "Issue Navigator - Hillel IT School JIRA");
         return true;
     }
+    public void clickButtonChangeViews(){
```
Comment (Artem100) [inline src/test/java/pages/SearchPage.java]: Put the method on the next line (press 'Enter')

File: src/test/java/pages/SearchPage.java [hunk 3]
```diff
@@ -69,4 +73,11 @@ public boolean atRequiredPage()
...
+        $(".aui-buttons").click();}
+    public void clickDetailView(){
+        $("a.aui-list-item-link[data-layout-key='split-view'").click(); }
+    public void clickListView(){
+        $("a.aui-list-item-link[data-layout-key='list-view']").click(); }
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java]: put the "}" on the next line

File: src/test/java/pages/SearchPage.java [hunk 4]
```diff
@@ -69,4 +73,11 @@ public boolean atRequiredPage()
     { Assert.assertEquals(title(), "Issue Navigator - Hillel IT School JIRA");
         return true;
     }
+    public void clickButtonChangeViews(){
+        $(".aui-buttons").click();}
+    public void clickDetailView(){
+        $("a.aui-list-item-link[data-layout-key='split-view'").click(); }
+    public void clickListView(){
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java]: Put the method on the next line (press 'Enter')

File: src/test/java/pages/SearchPage.java [hunk 5]
```diff
@@ -69,4 +73,11 @@ public boolean atRequiredPage()
     { Assert.assertEquals(title(), "Issue Navigator - Hillel IT School JIRA");
         return true;
     }
+    public void clickButtonChangeViews(){
+        $(".aui-buttons").click();}
+    public void clickDetailView(){
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java]: Put the method on the next line (press 'Enter')

File: src/test/java/pages/SearchPage.java [hunk 6]
```diff
@@ -69,4 +73,11 @@ public boolean atRequiredPage()
...
+        $("a.aui-list-item-link[data-layout-key='split-view'").click(); }
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java]: put the "}" on the next line

File: src/test/java/tests/SearchJira.java [hunk 7]
```diff
@@ -132,16 +135,20 @@ public void EpmtyResultsIssue() {
...
         searchPage.selectEpicFilter();
-        sleep(1000);
-        $$(By.xpath(".//a[@data-issue-key='QAAUT6-5']/img[@alt='Epic']")).filter(Condition.visible);}
+        searchPage.clickFiterTypeIssue();
+        searchPage.clickButtonChangeViews();
+        searchPage.clickDetailView();
+
+        List<SelenideElement> listImg= $(".list-content").$$("img");
+        for (WebElement element: listImg) {
+            Assert.assertEquals(element.getAttribute("alt"), "Epic");}
```
Comment (IgorLitviakov) [inline src/test/java/tests/SearchJira.java]: put the "}" on the next line

File: src/test/java/pages/SearchPage.java [hunk 8]
```diff
@@ -69,4 +73,11 @@ public boolean atRequiredPage()
     { Assert.assertEquals(title(), "Issue Navigator - Hillel IT School JIRA");
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java:74]: Put this curly bracket up to one row (after parentheses).

File: src/test/java/pages/SearchPage.java [hunk 9]
```diff
@@ -36,6 +36,11 @@ public void selectProjectQAAUTO6(String request){
         $(By.id("searcher-pid-input")).setValue(request);
         $(By.cssSelector("label[title='QAAUTO-6']")).click(); }
 
+    public void selectProject(String request){
+        $(By.cssSelector(".criteria-selector.aui-button.aui-button-subtle.drop-arrow")).click();
+        $(By.id("searcher-pid-input")).setValue(request);
+        $(By.cssSelector("label[title='" +request+"']")).click(); }
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java:42]: put this "}" on the next line

File: src/test/java/pages/SearchPage.java [hunk 10]
```diff
@@ -36,6 +36,11 @@ public void selectProjectQAAUTO6(String request){
         $(By.id("searcher-pid-input")).setValue(request);
         $(By.cssSelector("label[title='QAAUTO-6']")).click(); }
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java:37]: put the "}" on the next line

File: src/test/java/pages/SearchPage.java [hunk 11]
```diff
@@ -67,6 +72,15 @@ public void  clickNewFilterButton(){
 
     public boolean atRequiredPage()
     { Assert.assertEquals(title(), "Issue Navigator - Hillel IT School JIRA");
-        return true;
-    }
+        return true; }
+
+    public void clickButtonChangeViews(){
+        $(".aui-buttons").click();}
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java:78]: put the "}" on the next line

File: src/test/java/pages/SearchPage.java [hunk 12]
```diff
@@ -67,6 +72,15 @@ public void  clickNewFilterButton(){
...
+
+    public void clickDetailView(){
+        $("a.aui-list-item-link[data-layout-key='split-view'").click(); }
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java:81]: put the "}" on the next line

File: src/test/java/pages/SearchPage.java [hunk 13]
```diff
@@ -67,6 +72,15 @@ public void  clickNewFilterButton(){
...
+
+    public void clickListView(){
+        $("a.aui-list-item-link[data-layout-key='list-view']").click(); }
```
Comment (IgorLitviakov) [inline src/test/java/pages/SearchPage.java:84]: put the "}" on the next line

**Review by IgorLitviakov — CHANGES_REQUESTED** [review]

**Comment by SlavaKlepikov** [comment]
Mentioned remarks were fixed

**Review by IgorLitviakov — APPROVED** [review]

**Comment by Artem100** [comment]
Try to merge with master

**Pull request merged**